### DISCUSSION TITLE
Disable individual vault export

### DISCUSSION THREAD

**Discussion by jobritz** [dani-garcia/vaultwarden#4508] [Ideas]
Hi,

I already noticed, that the Bitwarden Enterprise Policy to disallow individual vault export can not be
implemented due to license issues.
But I still think, that this is an important feature which maybe could be implemented in the admin-page.

**Comment by tknx** [comment]
I'd like to add this as a requirement. My kids aren't allowed to have private passwords yet. And most companies I work for wouldn't allow it either.

Why distinguish? When I create an account, why does the "My Vault" not just be a shareable vault?

**Reply by stefan0xC** [reply]
@tknx the request is for [disallowing the individual vault export](https://bitwarden.com/help/policies/#remove-individual-vault-export) not disabling a private vault, which you can do by having your kids join an organization with the [remove individual vault policy](https://bitwarden.com/help/policies/#remove-individual-vault) enabled:
![image](https://github.com/user-attachments/assets/ebb84582-a0f9-48de-aa9a-b440886da401)
